src/ngx_http_encrypted_session_module.c

```diff
@@ -316,6 +326,70 @@ ngx_http_encrypted_session_expires(ngx_conf_t *cf, ngx_command_t *cmd,
 }
 
 
+static void
+ngx_http_encrypted_session_free_cipher_ctx(void *data)
+{
+    ngx_http_encrypted_session_main_conf_t      *esmcf;
```

Comment:
We can just write `xxx     *esmcf = data;` here according to nginx's convention :)